scipy/linalg/tests/test_decomp.py

```diff
@@ -2146,6 +2146,39 @@ def test_diff_input_types(self):
         ret = ordqz(self.B2, self.A1, sort='lhp')
         self.check(self.B2, self.A1, 'lhp', *ret)
 
+class TestOrdQZWorkspaceSize(TestCase):
+
+    def setUp(self):
+        seed(12345)
+
+    def test_decompose(self):
+
+        import numpy.random
+        from scipy.linalg import ordqz
```

Comment:
These imports are not needed here because both are imported already at the module level.